stdlib.go

```diff
@@ -128,24 +141,29 @@ func (a StdlibAdapter) handleMessagePrefix(msg string) string {
 }
 
 const (
-	logRegexpDate = `(?P<date>[0-9]{4}/[0-9]{2}/[0-9]{2})?[ ]?`
-	logRegexpTime = `(?P<time>[0-9]{2}:[0-9]{2}:[0-9]{2}(\.[0-9]+)?)?[ ]?`
-	logRegexpFile = `(?P<file>.+?:[0-9]+)?`
-	logRegexpMsg  = `(: )?(?P<msg>(?s:.*))`
+	stdlibRegexpPatternDate = `(?P<date>[0-9]{4}/[0-9]{2}/[0-9]{2})?[ ]?`
+	stdlibRegexpPatternTime = `(?P<time>[0-9]{2}:[0-9]{2}:[0-9]{2}(\.[0-9]+)?)?[ ]?`
+	stdlibRegexpPatternFile = `(?P<file>.+?:[0-9]+)?`
+	stdlibRegexpPatternMsg  = `(: )?(?P<msg>(?s:.*))`
 )
 
 var (
-	logRegexp = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	// StdlibRegexpFull captures date, time, caller (file), and message from stdlib log messages.
+	StdlibRegexpFull = regexp.MustCompile(stdlibRegexpPatternDate + stdlibRegexpPatternTime + stdlibRegexpPatternFile + stdlibRegexpPatternMsg)
+	// StdlibRegexpDefault
+	//captures date, time, caller (file) and message from stdlib log messages.
+	StdlibRegexpDefault = regexp.MustCompile(stdlibRegexpPatternDate + stdlibRegexpPatternTime + stdlibRegexpPatternMsg)
```

Comment:
The comment here is inconsistent with the declaration. Should it include stdlibRegexpPatternFile?

Reply:
No. file is not included on the default flags.

Source: https://pkg.go.dev/log#pkg-constants. Only date and time is included by default.

This is initial root cause for this PR. Because StdlibAdapter expects file by default while it's not enabled by default..

Reply:
I fixed the comment.